modules/oadp-monitoring.adoc

```diff
@@ -0,0 +1,383 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/troubleshooting.adoc
+
+:_content-type: CONCEPT
+[id="oadp-monitoring_{context}"]
+= OADP Monitoring
+
+The {product-title} provides a xref:../../monitoring/monitoring-overview.adoc#about-openshift-monitoring[monitoring stack] that allows users and administrators to effectively monitor and manage their OpenShift clusters, as well as monitor and analyze the workload performance of user applications and services running on the clusters, including receiving alerts if an event occurs.
+
+The OADP Operator leverages an OpenShift xref:../../monitoring/enabling-monitoring-for-user-defined-projects.adoc#enabling-monitoring-for-user-defined-projects_enabling-monitoring-for-user-defined-projects[User Workload Monitoring] provided by the OpenShift Monitoring Stack for retrieving metrics from the Velero service endpoint. The monitoring stack allows creating user-defined Alerting Rules or querying metrics, using the OpenShift Metrics query front-end.
+
+With enabled User Workload Monitoring it is possible to configure and use any Prometheus-compatible third-party UI, such as Grafana to visualize Velero metrics.
+
+[id="OADP-monitoring-setup_{context}"]
+== OADP monitoring setup
+Monitoring metrics requires enabling monitoring for the user-defined projects and creating a ServiceMonitor resource to scrape those metrics from the already enabled OADP service endpoint that resides in the `openshift-adp` namespace.
+
+[NOTE]
```

Comment:
1. I am not sold on adding this NOTE in the official documentation. The configuration files in the source code of the project are mostly for the development and users who follow docs within the project source code and not the official OCP docs.

2.  The instructions below also enables the user workload monitoring, however on the line 11 there of this file there is link to already existing procedure to enable user workload monitoring and lives in the *same* "home, so I wonder if the entire "OADP monitoring setup" should be just a paragraph here with first sentence:
   ```
Monitoring metrics requires enabling monitoring for the user-defined projects
and creating a ServiceMonitor resource to scrape those metrics from the
already enabled OADP service endpoint that resides in the openshift-adp namespace.
```

Plus some Prerequisites, with Procedure as a link to the:
 ```
xref:../../monitoring/monitoring-overview.adoc#enabling-monitoring-for-user-defined-projects_enabling-monitoring-for-user-defined-projects[Enabling monitoring for user-defined projects]
```

The source code contains that procedure as it's outside of the Red Hat OpenShift Documentation, however here I see duplication.


NOTE, that maybe it's better to keep it here as the few lines gives end to end instructions that are crafted for OADP and the user won't need to jump between different documentation parts and it's just few lines of docs that duplicates the instructions.

Reply:
@mpryc 
Thank you for your feedback. I made some changes based on your feedback.